articles/multifactor-authentication/guardian/configuration.md

```diff
@@ -0,0 +1,63 @@
+# Setting Up Guardian
+
+Guardian is Auth0's multifactor authentication (MFA) application that provides a simple, safe way for you to implement MFA. The Guardian app is currently available for mobile devices running iOS or Android.
+
+For applications where Guardian MFA is enabled, the user will be required to sign in **and** confirm the logon with a verified mobile device. You can find additional information on the user log in process [here](#).
+
+## Implementing Multifactor Authentication
+Within Auth0, you may implement MFA via the **Multifactor Auth** page of the Management Dashboard.
+
+![](/media/articles/mfa/guardian-dashboard.png)
+
+> Auth0 provides [built-in support](https://auth0.com/docs/multifactor-authentication#using-auth0-s-built-in-support) for MFA using Google Authenticator or Duo. You may choose to use either of these providers, in lieu of Guardian, on the **Multifactor Auth** page of the Management Dashboard.
+
+### Configuring Guardian in the Management Dashboard
+
+The first thing you will do when setting up Guardian is to decide whether you would like MFA to occur via push notifications, SMS, or both.
+
+* **Push Notifications**: the user receives, via the Guardian app, a push notification that requires their input prior to gaining access to the app;
```

Comment:
I'd also state that this option also enables user to use any code generator app (e.g. Google Authenticator)